R/utils.R

```diff
@@ -182,3 +182,117 @@ is_installed <- function(path_pkg = ".") {
 is_windows <- function() {
   identical(.Platform$OS.type, "windows")
 }
+
+#' Pin asset directory
+#'
+#' Add directories that need to be available on both branches when
```

Comment:
I think `Add` is no longer correct here. Maybe `capture`? I can adapt both help files later to that effect

Reply:
Maybe `pin` like in the function name?` I think that communicates well what is being done.